tests/README.md

```diff
@@ -1,8 +1,50 @@
 Unit tests
 ==========
 
+**WARNING**: These tests require an active Kafka cluster and will create new topics.
+
 From top-level directory run:
 
     $ tox
 
 **NOTE**: This requires `tox` ( please install with `pip install tox` ) and several supported versions of Python.
+
+Integration tests
+=================
+
+To run all of the integration test `modes` uncomment the following line from `tox.ini` and add the addresses to your Kafka and Confluent Schema Registry instances.
+
+    #python examples/integration_test.py {posargs} [yourhost:yourport] confluent-kafka-testing http://yourhost:yourport
```

Comment:
yourhost:yourport is not descriptive, use: `<bootstrap-servers>` and `[<schema-registry-url>]`

Note:  in shell syntax `<..>` means a variable field, `[]` means optional. `[foo]` thus means an optional literal "foo", while `[<address>]` means an optional field taking an address